vgen/src/main/java/io/gapi/vgen/go/GoLanguageProvider.java

```diff
@@ -50,7 +50,7 @@
 
   @SuppressWarnings("unchecked")
   public <Element> GeneratedResult generate(
-      Element element, SnippetDescriptor snippetDescriptor, GoGapicContext context) {
+      Element element, SnippetDescriptor snippetDescriptor, Object context) {
```

Comment:
In Java, a "JavaContext" ancestor was created (as an interface) for the equivalent situation.